tests/configuration_wrapper_ut/configuration_wrapper_ut_helpers.h

```diff
@@ -0,0 +1,436 @@
+// Copyright (C) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#ifndef CONFIGURATION_WRAPPER_UT_HELPERS_H
+#define CONFIGURATION_WRAPPER_UT_HELPERS_H
+
+#ifdef __cplusplus
+#include <cinttypes>
+#else
+#include <inttypes.h>
+#include <stdbool.h>
+#include <wchar.h>
+#endif
+
+#include "windows.h"
+#include "fabricruntime.h"
```

Comment:
Don't we need that #define to make sure we use the C COM interfaces anymore?

Reply:
We don't seem to have it in most other places, but I don't know if that's just because those are all only built with C.

All the code in fabricruntime.h is:

```
#if defined(__cplusplus) && !defined(CINTERFACE)
```

I suppose it kind of makes sense to define it in this case (well, in what is now called configuration_wrapper.h) in case someone tried to implement a config from a .cpp file (which generates some code with the extra `->lpVtbl` that won't work).

Not needed in the tests for sure though.

Reply:
I think I'll leave it without for now, consuming code could define CINTERFACE if needed (e.g. because it's built as c++).